<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-B1UU0SL" failures="1" tests="2" name="Test Cases" time="0.124" errors="0" timestamp="2022-01-29T12:57:38 EST">
  <testcase classname="com.testngannotation.TestClass" name="testCase1" time="0.021"/>
  <testcase classname="com.testngannotation.TestClass" name="testCase2" time="0.034">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.testngannotation.TestClass.testCase2(TestClass.java:13)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- testCase2 -->
</testsuite> <!-- Test Cases -->
